gradle.properties

```diff
@@ -2,9 +2,9 @@ org.gradle.caching=true
 group=io.ballerina.stdlib
 version=2.3.1-SNAPSHOT
```

Comment:
We need to bump the version to the next minor version right?

Reply:
@ravinperera00 can you please check?